base/client.jl

```diff
@@ -137,6 +137,12 @@ function repl_cmd(cmd, out)
 end
 
 function display_error(io::IO, er, bt)
+    if !isempty(bt)
+        st = stacktrace(bt)
+        if !isempty(st)
+            io = redirect(io, log_error_to, st[1])
+        end
+    end
```

Comment:
@stevengj by the way, here's an answer to your question about `log_error_to` - for the moment I've reinstated it exactly as it was used inside `display_error`. I think it was a bit of a misfeature if used to capture REPL errors, but it's easy to preserve the behavior for now.

Reply:
The point is, this will get removed when the deprecation is removed in 1.0, right?

Reply:
Right.  The user should get a depwarn if they ever manage to populate `log_error_to`.